packages/block-editor/src/components/block-pattern-picker/layout-setup-toolbar.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import {
+	chevronRight,
+	chevronLeft,
+	grid,
+	stretchFullWidth,
+} from '@wordpress/icons';
+
+function LayoutSetupToolbar( {
```

Comment:
I think we should call this "PatternSetup" or "SetupWithPattern" going forwards. Layout seems to introduce another concept that we might not want in all cases.

Reply:
I agree. I had initially named `BlockPatternPicker` something with `layout` and didn't rename all things to get the first feedback of how it feels.

>Don't mind the styling and names/code too much, as it will be refactored.